cmd/fluxd/main.go

```diff
@@ -127,6 +127,7 @@ func main() {
 		// syncing
 		syncInterval = fs.Duration("sync-interval", 5*time.Minute, "apply config in git to cluster at least this often, even if there are no new commits")
 		syncGC       = fs.Bool("sync-garbage-collection", false, "experimental; delete resources that were created by fluxd, but are no longer in the git repo")
+		dryGC        = fs.Bool("dry-run-garbage-collection", false, "experimental; log deletion process for resources that were created by fluxd, but are no longer in the git repo")
```

Comment:
If you make this start with `sync-garbage-collection-` it will sort next to the closely-related `sync-garbage-collection` in usage and docs and so on.

Reply:
The usage text implies that deletion will still happen. Perhaps something like `"experimental; only log what would be garbage collected, rather than deleting. Implies --sync-garbage-collection"`

Reply:
> If you make this start with `sync-garbage-collection-` it will sort next to the closely-related `sync-garbage-collection` in usage and docs and so on.

You mean in the sorting of names for options to start with? Didn't know that. How would you like `sync-garbage-collection-dry`

> The usage text implies that deletion will still happen. Perhaps something like `"experimental; only log what would be garbage collected, rather than deleting. Implies --sync-garbage-collection"`

You're absolutely right. I would just take your suggestion then.